### PR TITLE
Improved fastlane landing page docs

### PR DESCRIPTION
**Pull request by KrauseFx**
Need more growth hacking

### CODE REVIEW

**Comment by KrauseFx** [comment]
Ready to be merged from my side

File: README.md [hunk 1]
```diff
@@ -73,10 +75,12 @@ If you want to take a look at a project already using `fastlane`, check out the
 
 The setup assistant will create all the necessary files for you, using the existing app metadata from iTunes Connect.
```
Comment (lacostej) [inline README.md]: from iTunes Connect or Play Store :)

Comment (hemal) [inline README.md]: I think Google calls it "Google Play" instead of "Play Store".

**Comment by hemal** [comment]
After the proposed addition of "Google Play", it lgtm

**Comment by KrauseFx** [comment]
Thanks everyone, I'll update the PR tomorrow morning :+1:

**Comment by KrauseFx** [comment]
Done, merging as soon as the build is passing 🚀

**Pull request merged**